packages/api-derive/src/balances/account.ts

```diff
@@ -23,13 +23,19 @@ type DeriveCustomAccount = ApiInterfaceRx['derive'] & {
   }
 }
 
+function zero (api: ApiInterfaceRx) {
+  return api.registry.createType('Balance');
+}
+
 function getBalance (api: ApiInterfaceRx, [freeBalance, reservedBalance, frozenFee, frozenMisc]: BalanceResult): DeriveBalancesAccountData {
+  const votingBalance = api.registry.createType('Balance', freeBalance.toBn());
+
```

Comment:
```suggestion
```

Just reverting this. It is certainly a bit cleared, but at the same time since it is not used anywhere, guess it doesn't need to create a variable. (Although the optimizing compiler should take care of it...)

(Goes with the suggestion just below)